### PR TITLE
Sizrs

### PR DESCRIPTION
**Pull request by prbush**
Merging SIZRS deployment config finalized on 6/1/2023 with main.

### CODE REVIEW

**Comment by prbush** [comment]
Approving my own pull request.

**Review by prbush — COMMENTED** [review]
Approving own pull request.

File: .gitignore [hunk 1]
```diff
@@ -51,7 +51,7 @@ Module.symvers
 Mkfile.old
 dkms.conf
 
-# STM#@CubeIde Files
+# STM32CubeIde Files
```
Comment (prbush) [inline .gitignore:54]: Good.

**Review by prbush — COMMENTED** [review]
Good.

**Review by philbush5 — APPROVED** [review]
Approving pull request to merge SIZRS deployment config with main.

**Pull request merged**